src/ui/events.js

```diff
@@ -103,12 +102,40 @@ export class MapMouseEvent extends Event {
 }
 
 /**
- * `MapTouchEvent` is the event type for touch-related map events.
+ * `MapTouchEvent` is the type for objects emitted by touch-related [`Map` events](https://docs.mapbox.com/mapbox-gl-js/api/map/#map-events).
  * @extends {Object}
+ * @example
+ * // Example of a MapTouchEvent of type "touch"
+ * {
+ *     lngLat: {
+ *         lng: 40.203,
+ *         lat: -74.451
+ *     },
+ *     lngLats: [
```

Comment:
we could add another coord pair here as well as for `points` to drive home that some touch events have multiple touch points, but 1) i was hesitant to make the example too long and 2) i couldn't figure out a great way to emulate multi-touch in the browser and didn't want to misrepresent the actual event by accident.

Reply:
I agree on keeping the example as short as possible, but including multiple coordinate pairs if that makes the example more clear. Hopefully the GL JS team can help us present these objects clearly so users understand what they're looking at when they visit this doc.